gbpservice/tests/contrib/diskimage-create/disk_image_create.py

```diff
@@ -35,16 +33,16 @@ def parse_json(j_file):
 def get_nfp_branch_name_for_docker(file_path):
     data = file(file_path)
 
-    for line in data:                                                              
+    for line in data:
         if 'GBPSERVICE_BRANCH' in line:
             data.close()
             return line.split('=')[1].rstrip()
 
 
 def set_nfp_git_branch(nfp_branch_name, configurator_dir):
     Dockerfile_path = configurator_dir + '/Dockerfile'
-    cmd = "sudo sed -i \"s/mitaka_21st_march_base/%s/g\" %s" % (
-                                            nfp_branch_name, Dockerfile_path)
+    cmd = "sudo sed -i \"s/GIT-BRANCH-NAME/%s/g\" %s" % (
+               nfp_branch_name.replace('/', '\/'), Dockerfile_path)
```

Comment:
I think we can avoid doing "sed" for gbp branch name. The branch name can be passed to "docker build" command when the branch name is passed as an environment variable inside Dockerfile.
It's cleaner that way.